tests_integration/nativeapp/test_open.py

```diff
@@ -67,31 +67,27 @@ def test_nativeapp_open(
 
 
 @pytest.mark.integration
-@mock.patch.dict(
-    os.environ,
-    {
-        "SNOWFLAKE_CLI_FEATURES_ENABLE_PROJECT_DEFINITION_V2": "true",
-    },
-)
+@enable_definition_v2_feature_flag
 @mock.patch("typer.launch")
+@pytest.mark.parametrize("project_definition", ["v1", "v2"])
 def test_nativeapp_open_v2(
     mock_typer_launch,
     runner,
-    snowflake_session,
+    project_definition,
```

Comment:
is this version or name? can we clarify in the name?

Reply:
This is the parametrized value, either `"v1"` or `"v2"`.
Will "definition_version" be clearer?

Reply:
yes sure. thx

Reply:
done